lib/pat.c

```diff
@@ -5990,18 +5990,17 @@ pat_key_defrag_each(grn_ctx *ctx,
 }
 
 /* See test/command/suite/defrag/pat/README.md when you change this.
- * You must update tests for this too. */
+ * You must update tests for this too.
+ * Table locking is required when using this function. */
```

Comment:
```suggestion
 * If you're using a grn_pat in multi-thread/multi-process environment, you must use grn_obj_defrag() not
 * instead of using this function directly. */
```